src/System.Security.Permissions/src/System/Security/SecurityElement.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.s
+
+namespace System.Security
+{
+    public sealed partial class SecurityElement
+    {
+        public SecurityElement(string tag) { }
+        public SecurityElement(string tag, string text) { }
+        public System.Collections.Hashtable Attributes { get; set; }
+        public System.Collections.ArrayList Children { get; set; }
+        public string Tag { get; set; }
+        public string Text { get; set; }
+        public void AddAttribute(string name, string value) { }
+        public void AddChild(System.Security.SecurityElement child) { }
+        public string Attribute(string name) { return default(string); }
+        public System.Security.SecurityElement Copy() { return default(System.Security.SecurityElement); }
```

Comment:
My preference in this case would be to return ```this```.  Basically, do minimal work to get some reasonable result.  Returning ```this``` is no more work than returning null (which is what ```default(SecurityElement)``` is, and it makes the implementation ever-so-slightly more robust (if folks are calling this, they would not expect a null result).  In general, as well, I'd prefer it if we not use ```default(T)``` in the src implementation, instead returning ```this``` or ```null``` for reference types and the equivalent default value for primitive types, e.g. ```false``` for bool, ```0``` for int, etc.  Doing so just makes the code simpler and easier to read.